docs/user/reference/action.md

```diff
@@ -3,19 +3,49 @@
 
 > See also: {ref}`manage-actions`
 
-In Juju, an **action** is a script that is triggered via the {ref}`juju-cli` and applied to a {ref}`unit <unit>`. It contains a list of commands defined by a  {ref}`charm <charm>` to allow a {ref}`user <user>` with the right {ref}`access level <user-access-levels>` to interact with an {ref}`application <application>` in ways specific to the application. This may include anything from creating a snapshot of a database, adding a user to a system, dumping debug information, etc.
+An action is used to perform a named, parameterised operation and report back the results of said operation.
+It is triggered via the {ref}`juju-cli` and applied to one or more {ref}`units <unit>`.
+
+Actions are defined by a  {ref}`charm <charm>` to allow a {ref}`user <user>` with the right {ref}`access level <user-access-levels>` to interact with an {ref}`application <application>` in ways specific to the application.
```

Comment:
I think this needs to be in the first paragraph, otherwise the first paragraph, although it does clarify how think of it conceptually, doesn't really say much of relevance to the user. But I don't want to block/delay your PR on that -- that's the sort of stuff I can take care of myself.

Reply:
I've moved this to the first paragraph and tweaked the following paragraph.